console/console-init/ui/src/modules/iot-certificates/components/IoTCertificate/IoTCertificate.tsx

```diff
@@ -9,33 +9,43 @@ import {
   CertificateForm,
   CertificateCard
 } from "modules/iot-certificates";
+import { PageSection } from "@patternfly/react-core";
 
 export interface IIoTCertificateProps {
   certificate: IIoTCertificate;
+  onEdit: (certificate: IIoTCertificate) => void;
+  onDelete: (certificate: IIoTCertificate) => void;
+  onEnableOrDisable: (certificate: IIoTCertificate, isEnabled: boolean) => void;
   id: string;
 }
 
 export const IoTCertificate: React.FunctionComponent<IIoTCertificateProps> = ({
   certificate,
+  onEdit,
+  onDelete,
+  onEnableOrDisable,
   id
 }) => {
   const [onEditMode, setOnEditMode] = useState<boolean>(false);
 
   return (
-    <>
+    <PageSection style={{ paddingBottom: 0 }}>
       {onEditMode ? (
         <CertificateForm
           id={id}
           setOnEditMode={setOnEditMode}
           certificate={certificate}
+          onSave={onEdit}
```

Comment:
Here, I wonder child component event and parent event names are mismatch. can you revisit.
It should be both either onSave or onEdit.
Either onSave={onSave} or onEdit={onEdit}

Reply:
onSave is a function for onClick handler. It will be either edit or create event